eventbridge/README.md

```diff
@@ -0,0 +1,83 @@
+# AWS Eventbridge to Kinesis Firehose
+
+Terraform module which sets up a Kinesis Firehose delivery stream as a target for Eventbridge Events
+
+## Terraform versions
+
+Terraform 0.12 and newer. Submit pull-requests to `main` branch.
+
+## Usage
+
+```hcl
+data "aws_caller_identity" "current" {}
+
+resource "aws_cloudwatch_event_rule" "wildcard" {
+  name        = "wildcard"
+  description = "Capture all events in account"
+  event_pattern = jsonencode({
+    "account" : [data.aws_caller_identity.current.account_id]
+  })
+}
+
+module "observe_kinesis_firehose" {
+  source = "github.com/observeinc/terraform-aws-kinesis-firehose?ref=main"
+
+  name             = "observe-kinesis-firehose"
+  observe_customer = var.observe_customer
+  observe_token    = var.observe_token
+}
+
+module "observe_firehose_eventbridge" {
+  source           = "github.com/observeinc/terraform-aws-kinesis-firehose//eventbridge"
```

Comment:
Is that supposed to be a double forward slash?

Reply:
Yes, that makes the submodule nature explicit. It's not required in most cases, but it protects you against the case where the module you're trying to load has a relative path (e.g `source = "../example"`). By using the double slash, you ensure the entire root module is checked out. See https://www.terraform.io/docs/language/modules/sources.html#modules-in-package-sub-directories